packages/paste-core/layout/media-object/stories/index.stories.tsx

```diff
@@ -95,6 +101,257 @@ export const ConstrainedWidth = (): React.ReactNode => {
   );
 };
 
+const CustomMediaObjectCollection: React.FC = () => {
+  return (
+    <Stack orientation="vertical" spacing="space60">
+      <CustomizableMediaObject as="div" verticalAlign="center">
+        <CustomizableMediaFigure as="div" spacing="space100">
+          <Box backgroundColor="colorBackgroundDestructiveWeak" size="size20" />
+        </CustomizableMediaFigure>
+        <CustomizableMediaBody as="div">
+          <Text as="p">Some media Object body text</Text>
```

Comment:
Since we may use these stories in the future in our theme-designer and doc site, can we come up with more appropriate text for these examples while we're at it please?

This is a really solid improvement too, thanks for cleaning this up!

Reply:
Yep. Changing this section up a little after chatting with Simon, but I'll make text edits to the other examples.